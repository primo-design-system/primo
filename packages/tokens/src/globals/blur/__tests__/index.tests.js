const blurTokens = require('../index.json');

describe('blur tokens', () => {
  test(': matches snapshot', () => {
    expect(blurTokens).toMatchSnapshot();
  });
});
