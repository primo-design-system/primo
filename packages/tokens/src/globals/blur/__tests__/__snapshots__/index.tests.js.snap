// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blur tokens : matches snapshot 1`] = `
Object {
  "blur": Object {
    "filter": Object {
      "100": Object {
        "value": "4px",
      },
      "200": Object {
        "value": "8px",
      },
      "300": Object {
        "value": "16px",
      },
      "400": Object {
        "value": "32px",
      },
      "500": Object {
        "value": "64px",
      },
      "background": Object {
        "100": Object {
          "value": "{blur.filter.100.value}",
        },
        "200": Object {
          "value": "{blur.filter.200.value}",
        },
        "300": Object {
          "value": "{blur.filter.300.value}",
        },
        "400": Object {
          "value": "{blur.filter.400.value}",
        },
        "500": Object {
          "value": "{blur.filter.500.value}",
        },
      },
    },
  },
}
`;
